packages/react-urql/examples/4-core-node/package.json

```
@@ -6,6 +6,6 @@
   "dependencies": {
     "isomorphic-fetch": "2.2.1",
     "graphql-tag": "2.10.1",
-    "wonka": "3.2.1"
+    "wonka": "4.0.10"
```

For the sake of removing maintenance requirements, how do folks feel about

```suggestion
    "wonka": "4"
```

or

```suggestion
    "wonka": "latest"
```

Also open to us having something like renovate update these examples every couple of weeks.

---

I guess it depends on @kitten's plans for handling breaking changes in `wonka`. Unless there's special circumstances, I think we should use `4.0.10` or `^4.0.10`.

---

I feel like no examples should use pinning or lockfiles. Given our limited dependencies a simple minor-range (so for wonka `^4.x.x`) is completely sufficient to make sure that the exampel continues to work.
It's more restrictive than `latest` (I mean, API may change after all), but will ensure that we don't have to constantly keep all examples up-to-date.

More specifically though, why is `wonka` even installed though in this example? Is it used outside of `urql`? If not it can be removed from the `package.json`

---

I've never used this syntax before but looking at the npm docs, I believe `"4"` is equivalent to `"^4.x.x"` (i.e. no pinning).

> More specifically though, why is wonka even installed though in this example? Is it used outside of urql? If not it can be removed from the package.json

Wonka used to be an Urql peer dependency right? I think that is why.

---

Good point about not having lock files in examples -- removed.

Re. why is `wonka` installed, `pipe` and `subscribe` are used here:

```
pipe(
  client.executeQuery(request),
  subscribe(({ data, error }) => {
    if (error) {
      console.warn('Error', error);
    }

    console.log('Data', JSON.stringify(data, ' ', 2));
  })
);
```

---

That example above could also be simplified to use our shorthands (`client.query(...).toPromise()`), which is I guess what we'd recommend for Node.js usage anyway :)